common/app/views/fragments/inlineJSBlocking.scala.html

```diff
@@ -66,18 +66,32 @@
 
     // ************* LOAD THE MAIN APP ASYNC *************
 
+    @loadApp = {
+        @if(conf.Configuration.assets.useHashedBundles) {
+            // Polyfill for async script
+            (function (document) {
+                var script = document.createElement('script');
+                script.src = '@Static("javascripts/app.js")';
+                var ref = document.getElementsByTagName('script')[0];
+                ref.parentNode.insertBefore(script, ref);
+            })(document);
+        } else {
+            @Html(common.Assets.js.curl);
+            require(['boot']);
+        }
+    }
 
-    @if(conf.Configuration.assets.useHashedBundles) {
-        // Polyfill for async script
-        (function (document) {
-            var script = document.createElement('script');
-            script.src = '@Static("javascripts/app.js")';
-            var ref = document.getElementsByTagName('script')[0];
-            ref.parentNode.insertBefore(script, ref);
-        })(document);
+    @if(ElectionAppDelay.isSwitchedOn) {
+        window.guardian.triggerLoadApp = function () {
+            @loadApp
+        };
+        var page = window.guardian.config.page;
+        if (page.contentType !== 'Interactive' ||
+              !page.atoms || page.atoms[0] !== '2016/general-election') {
```

Comment:
Can't we check the atom server side? That way we can leave the rest of the site as-is and modify this just for the specific interactive, right?